server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/SystemMailboxesProvider.java

```diff
@@ -28,4 +28,6 @@
 
 public interface SystemMailboxesProvider {
     Stream<MessageManager> listMailboxes(Role aRole, MailboxSession session) throws MailboxException;
+
+    MessageManager findMailbox(Role role, MailboxSession session) throws MailboxException;
```

Comment:
We tends to avoid default implementation in interfaces, as far as I remember, but yes, you are right.